prow/config.yaml

```diff
@@ -3263,7 +3276,7 @@ presubmits:
     agent: kubernetes
     context: pull-service-catalog-unit
     always_run: true
-    skip_report: true
+    skip_report: false
```

Comment:
described in the tide readme
https://github.com/kubernetes/test-infra/blame/174ead37f62bb2e3e23270a777717c7b7ac9a0a9/prow/cmd/tide/README.md